internal/fields/sql_test.go

```diff
@@ -273,3 +275,33 @@ func TestField_SupportsProtobuf(t *testing.T) {
 		assert.Equal(t, event, got.Elem().Interface())
 	})
 }
+
+func TestField_SupportsProtobufWithoutMarshalMethods(t *testing.T) {
+	t.Run("pointer", func(t *testing.T) {
```

Comment:
if there's no code shared between the tests, i'd prefer multiple Test_ methods to prevent the extra indentation from t.Run

Reply:
fair, I just copied the style from above to be consistent.  Can change.